src/otx/algorithms/classification/adapters/mmcls/datasets/pipelines/otx_pipelines.py

```diff
@@ -23,6 +23,44 @@ class LoadImageFromOTXDataset(load_image_base.LoadImageFromOTXDataset):
     """Pipeline element that loads an image from a OTX Dataset on the fly."""
 
 
+@PIPELINES.register_module()
+class LoadResizeDataFromOTXDataset(load_image_base.LoadResizeDataFromOTXDataset):
+    """Load and resize image & annotation with cache support."""
+
+    def _create_load_img_op(self, cfg: Dict) -> Any:
+        """Creates image loading operation."""
+        return build_from_cfg(cfg, PIPELINES)
+
+    def _create_resize_op(self, cfg: Optional[Dict]) -> Optional[Any]:
+        """Creates resize operation."""
+        if cfg is None:
+            return None
+        return build_from_cfg(cfg, PIPELINES)
+
+
+@PIPELINES.register_module()
+class ResizeTo(Resize):
```

Comment:
I'm not sure about the functionality of this class and little bit confused of the naming.
Does this class needed for caching?

Reply:
`ResizeTo` is my custom wrapper which calls `Resize` op only if the current size is different from expected size. `Resize` would interpolate 100x100 image to 100x100 image, for example.
It's just for efficiency. Not to resize again.

Reply:
For more clarification, it will be applied to det/seg. It was just included in this PR. Sorry for confusion ;)